src/Page/Guides.elm

```diff
@@ -23,7 +23,10 @@ view model =
 
         teaserList : List Page.GuideTeaser.GuideTeaser
         teaserList =
-            if List.length model.search > 0 then
+            if model.language == Welsh then
```

Comment:
I think this might overrule the search in welsh, maybe the search conditional needs to be first. It's quite hard to test for with only one element in the to search against.